libs/core/src/lib/checkbox/checkbox/checkbox.component.html

```diff
@@ -6,6 +6,13 @@
     [name]="name"
     [disabled]="disabled"
     [indeterminate]="isIndeterminate"
+    [attr.tabIndex]="tabIndex"
+    [attr.aria-label]="ariaLabel || null"
+    [attr.aria-labelledby]="ariaLabelledby"
+    [attr.aria-checked]="_getAriaChecked()"
+    [attr.aria-describedby]="ariaDescribedby"
+    [attr.aria-disabled]="ariaDisabled"
+    [attr.title]="title"
     [class.fd-checkbox--compact]="compact"
     [ngClass]="state ? 'is-' + state : ''"
     [ngModel]="isChecked"
```

Comment:
I see that `isChecked` is a getter method. You should change it to a boolean property, and update it in the `nextValue()` method ... do the same as `_getAriaChecked()` above.

Reply:
changing isChecked will introduce breaking change. I have made changes for _getAriaChecked().